core/src/main/java/io/grpc/internal/AbstractServerStream.java

```diff
@@ -177,6 +178,20 @@ public StatsTraceContext statsTraceContext() {
     return statsTraceCtx;
   }
 
+  /**
+   * A hint to the stream that specifies how many bytes must be queued before
+   * {@link StreamListener#onReady()} will be called. A stream may ignore this property if
```

Comment:
onReady() won't be called when this is reached. Instead `isReady()` will return `false`.